src/reporters/noop_reporter.js

```diff
@@ -11,14 +11,12 @@
 // or implied. See the License for the specific language governing permissions and limitations under
 // the License.
 
-import Span from '../span.js';
-
-export default class NoopReporter {
+export default class NoopReporter implements Reporter {
   name(): string {
     return 'NoopReporter';
   }
 
-  report(span: Span): void {}
+  report(span: any): void {}
```

Comment:
Is this necessary?

Reply:
It's not, but it allows the file to not import anything. I'll remove.